src/functions-reference/real-valued_basic_functions.qmd

```diff
@@ -1698,3 +1698,69 @@ Implementation of the $W_0$ branch of the Lambert W function, i.e., solution to
 `R` **`lambert_wm1`**`(T x)`<br>\newline
 Implementation of the $W_{-1}$ branch of the Lambert W function, i.e., solution to the function $W_{-1}(x) \exp^{W_{-1}(x)} = x$
 {{< since 2.25 >}}
+
+## Hypergeometric Functions {#hypergeometric-functions}
+
+Hypergeometric functions refer to a power series of the form
+\begin{equation*}
+_pF_q(a_1,...,a_p;b_1,...,b_q;z) = \sum_{n=0}^\infty \frac{(a_1)_n\cdot\cdot\cdot(a_p)_n}{(b_1)_n\cdot\cdot\cdot(b_q)_n} \frac{x^n}{n!}
```

Comment:
z shows up on left and not on right and x shows up on right but not on left, so I think these are meant to be the same symbol.

Reply:
Based on the documentation at https://www.boost.org/doc/libs/1_85_0/libs/math/doc/html/math_toolkit/hypergeometric/hypergeometric_pfq.html, you are correct